include/configs/ipa9131.h

```diff
@@ -340,5 +342,7 @@
 	"bootm_size=0x36F00000\0"
 
 #define CONFIG_BOOTCOMMAND STANDARD_BOOT_COMMAND
-
+#define CONFIG_CMD_IPA9131_TEST_BOARD
+#define CONFIG_CMD_IPA9131_GO_SECURE
```

Comment:
Hold on... is this valid in U-Boot?  Surely not.  Writing fuses is a microloader thing (and, specifically, a bootstrap microloader thing).